docs/setup-cluster/deploy-cluster/slurm/slurm-requirements.rst

```diff
@@ -202,6 +202,27 @@ interacts with Slurm, we recommend the following steps:
 Determined should function with your existing PBS configuration. To optimize how Determined
 interacts with PBS, we recommend the following steps:
 
+-  Enable PBS to store job history.
+
+   To ensure successful job completion detection by the HPC launcher, it is crucial to have the job
+   history enabled in PBS. In the absence of proper configuration, the HPC launcher would fail to
+   resolve the status or information of a job post-completion.
```

Comment:
Could this be shorter, for example:

`Job completion detection requires that the job history feature be enabled.`

Reply:
Sure